python/shark_turbine/dynamo/importer.py

```diff
@@ -667,12 +673,31 @@ def _import_argument(
         elif isinstance(arg, torch_fx.immutable_collections.immutable_list):
             return self._import_list_argument(loc, arg, expected_jit_type)
         elif type(arg) in LITERAL_CONVERTER_MAP._cache:
-            with loc:
-                arg_value = LITERAL_CONVERTER_MAP.lookup(type(arg))(arg, self, self._cc)
+            # promote scalars to tensor types as appropriate
+            if isinstance(expected_jit_type, torch.TensorType):
+                arg_value = self._import_scalar_as_tensor(loc, arg)
+            else:
+                with loc:
+                    arg_value = LITERAL_CONVERTER_MAP.lookup(type(arg))(
+                        arg, self, self._cc
+                    )
             return arg_value
         else:
             raise NotImplementedError(f"FIXME: Unsupported Node Argument: {arg}")
 
+    def _import_scalar_as_tensor(self, loc: Location, arg: NodeArgument) -> Value:
```

Comment:
I'm pretty sure we don't want to do this for very long as we can just emit scalar ops for IREE. But at least it is in one place to fix.

Reply:
This is actually addressing ops where the python frontend accepts a scalar argument but internally converts it to a tensor - so for example the `aten.div.Tensor_mode` op - doesn't have a scalar equivalent, though one option is to decompose this into two ops, a `div.Scalar` and then whatever is appropriate for the rounding